app/views/users/show.html.erb

```diff
@@ -1,4 +1,5 @@
 <div>
-<h2>User1</h2>
-<p>I'm user one</p>
-</div>
\ No newline at end of file
+  <h2>User1</h2>
+  <p>I'm user one</p>
+</div>
+<%= render partial: 'biouser', locals: { user: :id } %>
```

Comment:
For your `user_show` page, kindly ensure a method is implemented to render the 3 most recent posts as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png) and also, do well to implement the `see all posts` link that takes you to the [`users_post` index page](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png) as shown in the respective wireframe😉. and also do well to ensure your views matches the wireframe👍🏼.